.github/ISSUE_TEMPLATE/i-have-a-problem-with-rive-flutter.md

```diff
@@ -0,0 +1,59 @@
+---
+name: I have a problem with rive-flutter
+about: Tell us what's wrong
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<-- 
+Thanks for using Rive!
+
+If something isn't working like you expected, please:
+- Make sure you've updated to rive-flutter to the latest version:
+			https://pub.dev/packages/rive/versions
+- Check out our help center
+			https://help.rive.app/
+
+   If you found a bug or if you can't find what you're looking for 
```

Comment:
If you "think" you've found a bug may read a tad friendlier I defer to others...